recipes/doctest/2.3.5/test_package/CMakeLists.txt

```diff
@@ -3,9 +3,14 @@ project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
+conan_include_build_modules()
```

Comment:
This is not needed, as `conan_basic_setup()` already includes the call to `conan_include_build_modules()`

Reply:
Done.